nat46/modules/nat46-module.c

```diff
@@ -157,6 +198,21 @@ int create_nat46_proc_entry(void) {
 
 static int __init nat46_init(void)
 {
+#ifdef PROTO_NETLINK
+	struct netlink_kernel_cfg nl_cfg = {
+		.input = nat46_nl_recv_msg
+	};
+	printk("nat46: module (version %s) loaded.\n", NAT46_VERSION);
```

Comment:
Two places printing the exact same content is confusing. If anything, this place should print something about procfs support not present, please use netlink.

Reply:
Thanks for the feedback.  I'll keep your suggestion in mind as I update my patch.